app/src/test/java/org/simple/clinic/home/overdue/OverdueScreenControllerTest.kt

```diff
@@ -128,32 +117,21 @@ class OverdueScreenControllerTest {
 
   @Test
   fun `when screen is created then the overdue appointments must be displayed`() {
+    // given
     val overdueAppointments = listOf(
-        PatientMocker.overdueAppointment(riskLevel = OverdueAppointment.RiskLevel.HIGHEST),
-        PatientMocker.overdueAppointment(riskLevel = OverdueAppointment.RiskLevel.LOW),
-        PatientMocker.overdueAppointment(riskLevel = OverdueAppointment.RiskLevel.NONE)
+        PatientMocker.overdueAppointment(riskLevel = OverdueAppointment.RiskLevel.HIGHEST, phoneNumber = null),
+        PatientMocker.overdueAppointment(riskLevel = OverdueAppointment.RiskLevel.LOW, phoneNumber = null),
+        PatientMocker.overdueAppointment(riskLevel = OverdueAppointment.RiskLevel.NONE, phoneNumber = null)
```

Comment:
Passing `null` in `phoneNumber` doesn't seem to make any difference here.
Any reason for it?

Reply:
I think it got added automatically by the refactoring when I added the parameter to the method. I'll remove it.